shell/models/pod.js

```diff
@@ -15,6 +15,36 @@ export const WORKLOAD_PRIORITY = {
 };
 
 export default class Pod extends WorkloadService {
+  _os = undefined;
+
+  get inStore() {
+    return this.$rootGetters['currentProduct'].inStore;
+  }
+
+  set os(operatingSystem) {
+    this._os = operatingSystem;
+  }
+
+  get os() {
+    if (this._os) {
+      return this._os;
+    }
+
+    return this?.node?.status?.nodeInfo?.operatingSystem;
+  }
+
+  get node() {
+    try {
+      const schema = this.$store.getters[`cluster/schemaFor`](NODE);
+
+      if (schema) {
+        this.$dispatch(`find`, { type: NODE, id: this.spec.nodeName });
+      }
+    } catch {}
```

Comment:
Similar to the previous empty catch block, I strongly feel that it would be beneficial to at least log an error if this fails.

Reply:
It's not really an error state to log so I'd rather not throw it to the console but if I wrap it in an of block it'd probably convey that better which is what I would've done if I hadn't been copy/pasta'ing this bit from elsewhere.